src/components/TitleSubtitle/TitleSubtitle.css

```diff
@@ -0,0 +1,33 @@
+.section-title-black {
+    text-align: center;
+    display: block;
+    color: var(--color-text);
+    margin: 0 0 20px 0;
+    font-weight: bold;
+
+}
+
+h2 {
+    font-size: 72px;
+}
+
+
+.section-title-white {
```

Comment:
Nice, dus we hebben een witte en een zwarte variant. Beide declaraties hebben overlappende declaraties (zoals `text-align`) en declaraties waarin ze verschillen (zoals de kleur). Kun je een manier bedenken om de selectors te combineren, zodat je de declaraties niet dubbel hoeft op te schrijven?